UnityProject/Assets/Scripts/Managers/CargoManager.cs

```diff
@@ -49,13 +49,15 @@ public class CargoManager : MonoBehaviour
 
 		public bool CargoOffline = false;
 		public bool RandomBountiesActive = true;
-    private int lastTimeRecorded = 0;
+		private int lastTimeRecorded = 0;
 		private int randomBountyTimeCheck = 0;
 
 		[SerializeField, BoxGroup("Random Bounties")] private float checkForTimeCooldown = 50f;
 		[SerializeField, BoxGroup("Random Bounties")] private Vector2 randomTimeRangeForRandomBounty = new Vector2(320, 690);
 		[SerializeField, BoxGroup("Random Bounties")] private List<CargoBounty> randomBountiesList = new List<CargoBounty>();
 
+		private List<int> randomJunkPrices = new List<int>();
```

Comment:
You can initialize the list once like this:
```suggestion
		private static readonly List<int> randomJunkPrices = new List<int> { 5, 10, 15 };
```